packages/widget_toolkit_pin/pubspec.yaml

```diff
@@ -19,13 +19,13 @@ dependencies:
   rx_bloc: ^6.0.0
   rx_bloc_list: ^5.0.0
   rxdart: ^0.28.0
-  shared_preferences: ^2.3.2
```

Comment:
As mentioned in the other PR, the dependency versions must be upgraded only if needed.

Reply:
Missed this one, changed